ui/app/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js

```diff
@@ -36,14 +34,12 @@ import {
   getCustomGasLimit,
   getCustomGasPrice,
   getDefaultActiveButtonIndex,
-  getEstimatedGasPrices,
-  getEstimatedGasTimes,
   getRenderableBasicEstimateData,
   getBasicGasEstimateBlockTime,
   isCustomPriceSafe,
   getTokenBalance,
   getSendMaxModeState,
-  getFastPriceEstimateInHexWEI,
+  getAveragePriceEstimateInHexWEI,
```

Comment:
Why change this to average?

Reply:
The values that we are now retrieving are different than the ones from ethgasstation. ethgasstation's api returns 'slow', 'average', 'fast' and 'fastest' estimates. Our new api, which is essentially a wrapper around gas now and etherscan, only returns three values: slow, average, fast. When the default to ethgasstation's fast option was set, it was because their 'average' was consistently too low for timely transaction completion. With our new estimates, the average should be sufficient for common use cases, and users can select fast when needed.